### ISSUE TITLE
How can we access the input_ids/attention mask in each train batch loop? 

### ISSUE THREAD

**Issue by tkmaker** [abhishekkrthakur/tez#42]
I tried using a train step callback but I am not sure how to get access to the dataloader input_ids and attention mask during each train step. Is this possible?
